ibmdbpy4nps/ae/client code examples/stock_prediction_modelscoring_wlm_nps.py

```diff
@@ -77,12 +84,12 @@
             temp_dict[column] = le
 
     wml_credentials = {
-                   'url': 'https://us-south.ml.cloud.ibm.com',
-                   'apikey':'xxxx'
+                   'url': 'https://eu-de.ml.cloud.ibm.com',
```

Comment:
Can we run this sample only on this cloud instance? Does it work as well from outside IBM? Is 'xxx' the real api key?